modules/monitoring-setting-up-metrics-collection.adoc

```diff
@@ -5,13 +5,13 @@
 [id="setting-up-metrics-collection_{context}"]
 = Setting up metrics collection
 
-To use the metrics exposed by your service, you need to configure OpenShift Monitoring to scrape metrics from the `/metrics` endpoint. You can do this using a ServiceMonitor, a custom resource definition (CRD) that specifies how a service should be monitored, or a PodMonitor, a CRD that specifies how a pod should be monitored. The former requires a Service object, while the latter does not, allowing Prometheus to directly scrape metrics from the metrics endpoint exposed by a Pod.
+To use the metrics exposed by your service, you need to configure OpenShift Monitoring to scrape metrics from the `/metrics` endpoint. You can do this using a ServiceMonitor, a custom resource definition (CRD) that specifies how a service should be monitored, or a PodMonitor, a CRD that specifies how a Pod should be monitored. The former requires a Service object, while the latter does not, allowing Prometheus to directly scrape metrics from the metrics endpoint exposed by a Pod.
```

Comment:
Noticing a couple of additional nits:

- you need to -> you must
- I'd rewrite the second sentence to clarify the CRD relationships; otherwise, it looks like a running list of, "You can do this using x, y, or z" (rather than "x or y"). So: "You can do this using a ServiceMonitor, _which is a_ custom resource definition (CRD) that specifies how a service should be monitored, or a PodMonitor, _which is a_ CRD that specifies how a Pod should be monitored."